src/frontend/planner_test/tests/testdata/output/row_filter.yaml

```diff
@@ -10,14 +10,14 @@
     select * from t where (v1,v2) > (1,2);
   batch_plan: |-
     BatchExchange { order: [], dist: Single }
-    └─BatchScan { table: t, columns: [t.v1, t.v2, t.v3], scan_ranges: [(t.v1, t.v2) > (Int32(1), Int32(2))], distribution: UpstreamHashShard(t.v1, t.v2, t.v3) }
+    └─BatchScan { table: t, columns: [t.v1, t.v2, t.v3], scan_ranges: [(t.v1, t.v2) >= (Int32(1), Int32(2))], distribution: UpstreamHashShard(t.v1, t.v2, t.v3) }
```

Comment:
In addition to planner test, let's add e2e tests for the issues fixed by this PR as well to verify the result is correct.